csg.yml

```diff
@@ -0,0 +1,52 @@
+hosts:
```

Comment:
this file should not be added to SEQLinkage repository. Please remove. You can put it in alzheimers-family analysis repo if you want.

Reply:
removed